src/gui/syncrunfilelog.cpp

```diff
@@ -145,7 +145,6 @@ void SyncRunFileLog::logItem( const SyncFileItem& item )
 
     const QChar L = QLatin1Char('|');
     _out << ts << L;
-    _out << QString::number(item._requestDuration) << L;
```

Comment:
you could keep two pipe chars, in case a script splits by '|' that still works, only the duration is empty.